app/javascript/data/advancedFilterOptions.js

```diff
@@ -178,7 +178,7 @@ export const advancedFilterOptions = [
     title: 'Preferred Contact Time (Select)',
     description: 'Monitoree preferred contact time',
     type: 'select',
-    options: ['Morning', 'Afternoon', 'Evening', ''],
+    options: ['Early Morning', 'Morning', 'Afternoon', 'Evening', 'Late Night', ''],
```

Comment:
I know it wasn't in the mock-ups, but do you think it would be helpful to have some indication of the allowed span of times? So like `Early Morning (Between Midnight and 08:00)` or along those lines. As it is now, it's not totally transparent which times = which options.

Reply:
that's a good point, maybe we can include a tooltip with that info

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1019/commits/daa0195b5441006dc8259d33d13da45f325d66c6#diff-9609a958c89218f92ffdf7d0c6635f81734d4951c7261a249248114a23fcd24dR184